### DISCUSSION TITLE
Is it possible to use this library for touch events

### DISCUSSION THREAD

**Discussion by kpp91302** [doronz88/pymobiledevice3#1221] [Q&A]
I'm trying to use this library to interact with the iphone gui(non-jailbroken device). more specifically to be able to press buttons like tab, search bar, etc in safari. is this possible without using appium?

**Comment by doronz88** [comment]
It isn't

**Reply by kpp91302** [reply]
is there a seperate lockdown or remoteXPC service like backboardd I could implement on my own to try to get this functionality or is this generally just not possible?

**Reply by doronz88** [reply]
Not that I know of
